Frontend/src/pages/login/login.css

```diff
@@ -1,13 +1,23 @@
 /* Google Fonts - Poppins */
-@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600&display=swap");
-
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600&display=swap');
+:root{
+    --black-20: rgb(20, 20, 20);
+    --gray-52: rgb(52, 52, 52);
+    --gray-72: rgb(72, 72, 72);
+    --gray-203: rgb(203, 213, 225);
+    --white-255: rgb(255, 255, 255);
+    --white-248: rgb(248, 250, 252);
+    --skyblue-190: hsl(190, 97%, 47%);
+    --darkblue-245: hsl(245, 95%, 22%);
+    --blue-208: hsl(208, 97%, 58%);
+}
```

Comment:
Welldone @Tammy-Ajoko  for this great work, i will suggest a few changes:

Rather than repeating the root color variables and the google font in each file(login and sign up), you could use 
`@import url(../../GlobalStyles.css);` to import the global CSS file to each of the files